src/NServiceBus.Core.Tests/ArgumentExceptionTests.cs

```diff
@@ -74,56 +74,68 @@ public void WriteAllPublicMembersWithNoArgumentChecking()
                     }
-                    if (!method.HasBody)
-                    {
-                        continue;
-                    }
-                    if (!method.IsPublic)
-                    {
-                        continue;
-                    }
-                    if (MethodCallSelf(method))
-                    {
-                        continue;
-                    }
-                    if (!MethodContainsArgumentException(method))
-                    {
-                        WriteMethod(method);
-                    }
-                }
-                foreach (var property in type.Properties)
-                {
-                    if (property.PropertyType.Name == "Boolean")
+                    if (method.Name.StartsWith("op_"))
                     {
                         continue;
                     }
-                    if (property.SetMethod == null)
+                    if (ContainsObsoleteAttribute(method))
                     {
                         continue;
                     }
-                    if (!property.SetMethod.HasBody)
+                    if (!method.HasBody)
                     {
                         continue;
                     }
-                    if (!property.SetMethod.IsPublic)
+                    if (!method.IsPublic)
                     {
                         continue;
                     }
-                    if (ContainsObsoleteAttribute(property))
+                    if (MethodCallSelf(method))
                     {
                         continue;
                     }
-
-                    if (!MethodContainsArgumentException(property.SetMethod))
+                    if (!MethodContainsArgumentException(method))
                     {
-                        WriteMethod(property.SetMethod);
+                        WriteMethod(method, stringWriter);
                     }
                 }
+                //foreach (var property in type.Properties)
+                //{
+                //    if (property.PropertyType.Name == "Boolean")
+                //    {
+                //        continue;
+                //    }
+                //    if (property.SetMethod == null)
+                //    {
+                //        continue;
+                //    }
+                //    if (!property.SetMethod.HasBody)
+                //    {
+                //        continue;
+                //    }
+                //    if (!property.SetMethod.IsPublic)
+                //    {
+                //        continue;
+                //    }
+                //    if (ContainsObsoleteAttribute(property))
+                //    {
+                //        continue;
+                //    }
+
+                //    if (!MethodContainsArgumentException(property.SetMethod))
+                //    {
+                //        WriteMethod(property.SetMethod, stringWriter);
+                //    }
+                //}
```

Comment:
This change was made by @danielmarbach so I'll let him comment on the intent here.